lib/vc.js

```diff
@@ -577,3 +580,18 @@ function _checkCredential(credential) {
       `"expirationDate" must be a valid date: ${credential.expirationDate}`);
   }
 }
+
+function _validateUriId({id, propertyName = 'id'}) {
```

Comment:
I wouldn't set a default here -- that "default" only makes sense in one particular use of the function and every other usage doesn't.

```suggestion
function _validateUriId({id, propertyName}) {
```